apps/sync/js/crypto/fxsyncwebcrypto.js

```diff
@@ -272,7 +271,7 @@ undle for collection ${collectionName}`);
     } catch(e) {
       return Promise.reject('Record cannot be JSON-stringified');
     }
-    const cleartext = StringConversion.rawStringToByteArray(cleartextStr);
+    const cleartext = StringConversion.utf16StringToByteArray(cleartextStr);
```

Comment:
Same as above, since the SyncStorage service always encodes BSOs in UTF8 (see https://docs.services.mozilla.com/storage/apis-1.5.html#basic-storage-object) we should not upload UTF16.